cmd/quarterdeck/main.go

```diff
@@ -256,47 +364,49 @@ findUnverified:
 
 	// Remove all unverified users
 	for _, user := range unverified {
-		if dry {
-			fmt.Printf("would remove user (%s) with email %s\n", user.ID.String(), user.Email)
-		}
-
 		// List all orgs for the user
 		var orgs []*models.Organization
 		if orgs, _, err = models.ListUserOrgs(ctx, user.ID, pagination.New("", "", 10)); err != nil {
-			err = errors.Join(err, err)
+			errs = errors.Join(errs, err)
 			continue
 		}
 
 		if len(orgs) == 0 {
-			err = errors.Join(err, fmt.Errorf("user (%s) has no organizations, why?", user.ID.String()))
+			errs = errors.Join(errs, fmt.Errorf("user (%s) has no organizations, why?", user.ID.String()))
 			continue
 		}
 
 		if len(orgs) > 1 {
-			err = errors.Join(err, fmt.Errorf("unverified user (%s) has more than one organization, why?", user.ID.String()))
+			errs = errors.Join(errs, fmt.Errorf("user (%s) has more than one organization, why?", user.ID.String()))
 			continue
 		}
 
 		if dry {
-			fmt.Printf("would remove org (%s) for user (%s)\n", orgs[0].ID.String(), user.ID.String())
+			fmt.Printf("would remove user (%s) with email %s\n", user.ID.String(), user.Email)
+			fmt.Printf("would remove org (%s) with name %s\n", orgs[0].ID.String(), orgs[0].Name)
 			continue
 		}
 
-		// Remove the user from the organization, this also deletes the user
-		if _, _, err = user.RemoveOrganization(ctx, orgs[0].ID, true); err != nil {
-			err = errors.Join(err, err)
+		// Delete the user
+		if err = deleteUser(ctx, user); err != nil {
+			errs = errors.Join(errs, err)
```

Comment:
Ran into a snag here, `RemoveOrganization` checks that the user is not the only member of the organization but in this case we also want to remove the organization. So I'm deleting the user here directly and also deleting the organization, which I think is fine because these are unverified users.